tests/conftest.py

```diff
@@ -33,11 +35,18 @@ def ensure_system(cmd):
 def pytest_addoption(parser):
     parser.addoption("--dvsname", action="store", default=None,
                       help="dvs name")
+    parser.addoption("--vctns", action="store", default=None,
+                     help="vct namespace")
+    parser.addoption("--topo", action="store", default=None,
+                     help="vct topology file")
     parser.addoption("--keeptb", action="store_true", default=False,
                       help="keep testbed after test")
     parser.addoption("--imgname", action="store", default="docker-sonic-vs",
                       help="image name")
 
+def random_string(size=2, chars=string.ascii_uppercase + string.digits ):
+   return ''.join( random.choice( chars ) for x in range( size ))
```

Comment:
four spaces?

Reply:
interface name's max length was a limitation to make it four spaces...will check if i can resize by renaming the interface name that goes into the chassisBridge.